engine/wwtlib/Graphics/Shaders.cs

```diff
@@ -1809,15 +1810,25 @@ public static void Init(RenderContext renderContext)
                 uniform float bscale;
                 uniform float min;
                 uniform float max;
+                uniform bool transparentBlack;
                 uniform int scaleType;
                 uniform float opacity;
                 float e = 2.71828182845904523536028747135266249775724709369995;
+                
+                bool isNaN(float value){
+                    return !(value < 0.0 || 0.0 < value || value == 0.0);
+                }
```

Comment:
Now you got me curious too...

Reply:
It works! At least on my GPU... I would stay with the more complex implementation however, because I've seen reports of some nvidia compiler "optimizing" away similar things.

Reply:
Even better, I now see I am allowed to use the built in isnan function now that we are using webgl 2.0 :)

Reply:
Never mind, strangely the builtin isnan doesn't seem to work properly. I reverted back to the original solution.